src/tools/rbd/Utils.cc

```diff
@@ -275,6 +297,96 @@ int get_special_pool_image_names(const po::variables_map &vm,
   return 0;
 }
 
+int get_special_pool_image_with_id_names(const po::variables_map &vm,
```

Comment:
I think you might be able to get away w/o duplicating this long method. In "group remove", if you detect that "--image-id" is populated, it doesn't make sense to support the "[pool/]image-name" positional